### ISSUE TITLE
error installing dependencies with yarn

### ISSUE THREAD

**Issue by CoreyCole** [bespoken/alexa-skills-kit-nodejs-lambda-boilerplate#2]
Trying to install node_modules with yarn, I get
```
bespoken-tools@0.9.25: The engine "node" is incompatible with this module. Expected version "4.3.2"
```

Not sure if this is important for you guys, but I thought I'd report. I tried running `npm upgrade`  and got
```
+ fuse.js@2.7.4
+ source-map-loader@0.1.6
+ dotenv@4.0.0
+ moment@2.22.1
+ alexa-sdk@1.0.25
+ es6-promise@4.2.4
+ source-map-support@0.4.18
+ bespoken-tools@0.9.25
+ isomorphic-fetch@2.2.1
+ aws-sdk@2.226.1
```
But, I still got the same error with yarn after upgrading.

**Comment by CoreyCole** [comment]
To fix, I just needed to update bespoken-tools dependency manually in package.json (PR #3)

**Comment by CoreyCole** [comment]
Working on updating all the dependencies in #3

**Comment by CoreyCole** [comment]
merged in #3
